magenta-lib/src/main/scala/magenta/deployment_type/Lambda.scala

```diff
@@ -22,6 +22,13 @@ object Lambda extends DeploymentType  {
       |As a result you can bundle stage specific configuration into the respective files.
     """.stripMargin
 
+  val regionsParam = Param[List[String]]("regions",
+    documentation = 
+      """
+      |One or more AWS region name where the lambda should be deployed - If this is not specified then `eu-west-1` will be used.
+      """.stripMargin
+  )
```

Comment:
`Param` has a `withDefault` method which would be a better place to deal with the `eu-west-1` default.